src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -29,7 +31,7 @@
      */
     private static final String CLIENT_TYPE = "client";
     private static final String COUNTRY_TYPE = "country";
-    private static final String NOTE_TYPE = "note";
+    private static final String CLIENT_NOTE_TYPE = "client note";
```

Comment:
This variable doesn't seem to be used, since you already removed the redundant code

Reply:
done!

Reply:
I think this variable still isn't being used